Gaijinizer patch/Map422.txt

```diff
@@ -21,10 +21,12 @@
 う、うそでしょ……！？
 
 ぼく、またこのすがた…！？
-> CONTEXT: Map422/events/2/pages/0/10/Dialogue/kazuya-c < UNTRANSLATED
-> CONTEXT: Map446/events/2/pages/0/16/Dialogue/kazuya-c < UNTRANSLATED
-> CONTEXT: Map466/events/4/pages/0/274/Dialogue/kazuya-c < UNTRANSLATED
+> CONTEXT: Map422/events/2/pages/0/10/Dialogue/kazuya-c
+> CONTEXT: Map446/events/2/pages/0/16/Dialogue/kazuya-c
+> CONTEXT: Map466/events/4/pages/0/274/Dialogue/kazuya-c
+N-No way...!
 
+I'm... back in this form again...!#Ron
 > END STRING
```

Comment:
Would the phrasing on this second line indicate confusion? (i.e, should we add a question mark here?)

Reply:
`I'm... back in this form again...?!?` sounds like the best, then.